packages/css-library/tokens/fonts.json

```diff
@@ -0,0 +1,30 @@
+{
+  "font": {
+    "family": {
+      "sans": { "value": "'Source Sans Pro', 'Helvetica Neue', Helvetica, Roboto, Arial, sans-serif" },
+      "serif": { "value": "Bitter, Georgia, Cambria, 'Times New Roman', Times, serif" }
+    },
+    "weight": {
+      "normal": { "value": "400"},
+      "bold": { "value": "700"}
+    },
+    "style": {
+      "normal": { "value": "normal" },
+      "italic": { "value": "italic" }
+    },
+    "size": {
+      "sm": { "value": "1.5rem" },
+      "base": { "value": "1.6rem" },
+      "md": { "value": "1.7rem" },
+      "lg": { "value": "2rem" },
+      "xl": { "value": "3rem" },
+      "2xl": { "value": "4rem" },
+      "h1": { "value": "{font.size.2xl}" },
+      "h2": { "value": "{font.size.xl}" },
+      "h3": { "value": "{font.size.lg}" },
+      "h4": { "value": "{font.size.md}" },
+      "h5": { "value": "{font.size.sm}" },
+      "h6": { "value": "{font.size.sm}" }
+    }
```

Comment:
I've matched these to our [current values for utility classes](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/40dbe6725a106534e08fe6b8e6d03a946a82986e/packages/formation/sass/utilities/_font-size.scss#L1-L14) rather than what [USWDS3 has listed](https://designsystem.digital.gov/design-tokens/typesetting/font-size/#theme-tokens). For example, we don't have `3xs`, `2xs`, and `xs` like USWDS does, and our `lg` token is 2rem/20px instead of 22px.